api/src/main/java/org/pmiops/workbench/cohortbuilder/ParticipantCounter.java

```diff
@@ -21,13 +21,18 @@
  * defined by the provided {@link SearchRequest}.
  */
 @Service
-public class SubjectCounter {
+public class ParticipantCounter {
 
     private static final String COUNT_SQL_TEMPLATE =
             "select count(distinct person_id) as count\n" +
                     "from `${projectId}.${dataSetId}.person` person\n" +
                     "where\n";
 
+    private static final String ID_SQL_TEMPLATE =
+            "select distinct person_id\n" +
```

Comment:
Distinct shouldn't be necessary. It's already unique in the person table. (Without it will be faster.)